tests/models/components/test_small_deconv_net.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import torch
+
+from src.models.components.small_deconv_net import SmallDeconvNet as cls
+
+
+class TestSmallDeconvNet:
+    def test__init__(self):
+        mod = cls(256, 256, 3, 256, True, torch.nn.ReLU())
+        assert mod.channels == 3
+        assert mod.fc_init.in_features == 256
+        assert mod.bias.shape == (3, 256, 256)
+        assert mod.nl == torch.nn.ReLU()
```

Comment:
等価表現は使えませんので、`isinstance`を用いましょう。
```suggestion
        assert isinstance(mod.nl, torch.nn.ReLU)
```